src/Blocks/custom/result-output-item/components/result-output-item-options.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { PanelBody, TextControl } from '@wordpress/components';
+import { checkAttr, getAttrKey, icons, IconLabel, Notification } from '@eightshift/frontend-libs/scripts';
+import manifest from '../manifest.json';
+
+export const ResultOutputItemOptions = ({
+	attributes,
+	setAttributes,
+}) => {
+
+	const resultOutputItemName = checkAttr('resultOutputItemName', attributes, manifest);
+	const resultOutputItemValue = checkAttr('resultOutputItemValue', attributes, manifest);
+
+	return (
+		<PanelBody title={__('Result Item', 'infobip')}>
```

Comment:
```suggestion
		<PanelBody title={__('Result Item', 'eightshift-forms')}>
```
Pls search-replace this string in project